copyandpush/copyandpush.test.js

```diff
@@ -9,3 +9,13 @@ describe('copyAndPush', () => {
     expect(actual).toEqual(expected);
```

Comment:
To test whether the original array has been mutated or not, add an additional `expect` statement here:
```js
expect(numbers).toEqual([1, 2, 3]);
```

It should pass, based on how your function is written, you just weren't quite testing the right thing in the other test.